diff --git a/dist/database/migrations/readMigrationFiles.js b/dist/database/migrations/readMigrationFiles.js
index 106178301693ddcd97e477940bfbb836d83ec529..9f0be6d26a7b043b814e1886a28c3a7d8af51442 100644
--- a/dist/database/migrations/readMigrationFiles.js
+++ b/dist/database/migrations/readMigrationFiles.js
@@ -19,7 +19,7 @@ import path from 'path';
     });
     return Promise.all(files.map(async (filePath)=>{
         // eval used to circumvent errors bundling
-        let migration = await eval(`${typeof require === 'function' ? 'require' : 'import'}('${filePath.replaceAll('\\', '/')}')`);
+        let migration = await eval(`${typeof require === 'function' ? 'require' : 'import'}('${filePath.replaceAll('\\', '/').replace(".ts", "")}')`);
         if ('default' in migration) migration = migration.default;
         const result = {
             name: path.basename(filePath).split('.')?.[0],
